src/main/java/com/sainttx/banknotes/command/BanknotesCommand.java

```diff
@@ -61,13 +62,27 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
                     sender.sendMessage(plugin.colorMessage(plugin.getConfig().getString("messages.invalid-number")));
                 } else {
                     ItemStack banknote = plugin.createBanknote(sender.getName(), amount);
-                    target.getInventory().addItem(banknote);
-                    target.sendMessage(plugin.colorMessage(plugin.getConfig().getString("messages.note-received")
-                            .replace("[money]", plugin.formatDouble(amount))
-                            .replace("[player]", sender.getName())));
-                    sender.sendMessage(plugin.colorMessage(plugin.getConfig().getString("messages.note-given")
-                            .replace("[money]", plugin.formatDouble(amount))
-                            .replace("[player]", target.getName())));
+                    
```

Comment:
You're fine, it's not like your solution is incorrect (it can just be simplified a tiny bit)!.

I really appreciate the help, I don't really have much time to manage some of my personal projects so PRs and feature implementations are always awesome. Thanks!